web/plugin-manager/src/components/toggleTheme.tsx

```diff
@@ -0,0 +1,28 @@
+import Toggle from "react-toggle";
+import moon from "../assets/pictos/moon.svg";
+import sun from "../assets/pictos/sunWhite.svg";
+import { UIStore } from "../store/UIStore";
+import "react-toggle/style.css"
+import "../index.css"
+
+const toggleTheme = () => {
+    UIStore.useState(s => (s.theme));
+    const handleChange = () => {
+        UIStore.update(s => {
+            s.theme = s.theme == "dark" ? "light" : "dark";
+        })
+    };
+    return (
+            <Toggle
+                defaultChecked={UIStore.useState(s => (s.theme == "light" ? true : false))}
+                icons={{
+                    checked: <img src={sun} />,
+                    unchecked: <img src={moon} />,
```

Comment:
duplicated code here, with another react project, is there a way to reuse component from one project to another?

Reply:
I can't send it to the shared folder because it's JSX. so for now it's okay to duplicate because it's generic. But yes we will merge to one folder when we rebrand (Soon)